### PR TITLE
add python solution for 691

### PR DESCRIPTION
**Pull request by jjliewie**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/usaco-691.mdx [hunk 1]
```diff
@@ -2,7 +2,7 @@
 id: usaco-691
 source: USACO Silver 2017 January
 title: Hoof, Paper, Scissors
-author: Óscar Garries (C++), Owen Wang (Java)
+author: Óscar Garries (C++), Owen Wang (Java), Juheon Rhee (Python)
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_hps_silver_jan17.html)
```
Comment (ryanchou-dev) [inline solutions/usaco-691.mdx:8]:
```suggestion
[Official Analysis (Java)](http://www.usaco.org/current/data/sol_hps_silver_jan17.html)
```

**Review by ryanchou-dev — APPROVED** [review]
⭐

File: solutions/usaco-691.mdx [hunk 2]
```diff
@@ -108,4 +108,34 @@ public class hps {
...
+		gesture = r.readline().strip()
+		if gesture == 'H':
+			h += 1
+		elif gesture == 'P':
+			p += 1
+		elif gesture == 'S':
+			s += 1
+		gestures.append([h, p, s])
+
+ans = 0
```
Comment (SansPapyrus683) [inline solutions/usaco-691.mdx:129]: rename to `max_wins`

**Review by SansPapyrus683 — APPROVED** [review]

File: solutions/usaco-691.mdx (hunk 2 above)
Comment (jjliewie) [reply solutions/usaco-691.mdx:129]: okay

**Review by SansPapyrus683 — APPROVED** [review]

**Pull request merged**